launch/tier4_control_launch/launch/control.launch.xml

```diff
@@ -0,0 +1,186 @@
+        <remap from="~/service/select_external_command" to="~/select_external_command"/>
+        <remap from="~/input/local/control_cmd" to="/api/external/set/command/local/control"/>
+        <remap from="~/input/local/shift_cmd" to="/api/external/set/command/local/shift"/>
+        <remap from="~/input/local/turn_signal_cmd" to="/api/external/set/command/local/turn_signal"/>
+        <remap from="~/input/local/heartbeat" to="/api/external/set/command/local/heartbeat"/>
+        <remap from="~/input/remote/control_cmd" to="/api/external/set/command/remote/control"/>
+        <remap from="~/input/remote/shift_cmd" to="/api/external/set/command/remote/shift"/>
+        <remap from="~/input/remote/turn_signal_cmd" to="/api/external/set/command/remote/turn_signal"/>
+        <remap from="~/input/remote/heartbeat" to="/api/external/set/command/remote/heartbeat"/>
+        <remap from="~/output/control_cmd" to="/external/selected/external_control_cmd"/>
+        <remap from="~/output/gear_cmd" to="/external/selected/gear_cmd"/>
+        <remap from="~/output/turn_indicators_cmd" to="/external/selected/turn_indicators_cmd"/>
+        <remap from="~/output/hazard_lights_cmd" to="/external/selected/hazard_lights_cmd"/>
+        <remap from="~/output/heartbeat" to="/external/selected/heartbeat"/>
+        <remap from="~/output/current_selector_mode" to="~/current_selector_mode"/>
+        <param from="$(var external_cmd_selector_param_path)"/>
+        <extra_arg name="use_intra_process_comms" value="$(var use_intra_process)"/>
+      </composable_node>
+
+      <!-- external cmd converter -->
+      <composable_node pkg="external_cmd_converter" plugin="external_cmd_converter::ExternalCmdConverterNode" name="external_cmd_converter">
+        <param from="in/external_control_cmd" to="/external/selected/external_control_cmd"/>
+        <param from="in/shift_cmd" to="/external/selected/gear_cmd"/>
+        <param from="in/emergency_stop" to="/external/selected/heartbeat"/>
+        <param from="in/current_gate_mode" to="/control/current_gate_mode"/>
+        <param from="in/odometry" to="/localization/kinematic_state"/>
+        <param from="out/control_cmd" to="/external/selected/control_cmd"/>
+        <param from="out/latest_external_control_cmd" to="/api/external/get/command/selected/control"/>
+        <param name="csv_path_accel_map" value="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/accel_map.csv"/>
+        <param name="csv_path_brake_map" value="$(find-pkg-share raw_vehicle_cmd_converter)/data/default/brake_map.csv"/>
+        <param name="ref_vel_gain" value="3.0"/>
+        <param name="wait_for_first_topic" value="true"/>
+        <param name="control_command_timeout" value="1.0"/>
+        <param name="emergency_stop_timeout" value="3.0"/>
+        <extra_arg name="use_intra_process_comms" value="$(var use_intra_process)"/>
+      </composable_node>
+    </node_container>
```

Comment:
[Question] Why did you change here from `IncludeLaunchDescription`?
https://github.com/autowarefoundation/autoware.universe/blob/0324e054fdf1e2ff903aa6d3faa4e6752dc11b9d/launch/tier4_control_launch/launch/control.launch.py#L199-L223

Reply:
Because it's more straightforward to write all the contents in xml, I think. There is only one `load_composable_nodes` in external_cmd_converter/selector python launch file.

Reply:
I think it's okay to replace it here. (It would be ideal to do it in another PR, though.)
Then, don't you need to remove the old `.py` files of external_cmd_converter/selector?